src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <div className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried && (
+        <p className="Person__partner">
+          {sex === 'm'
+            ? `${partnerName} is my wife`
+            : `${partnerName} is my husband`}
```

Comment:
Suggestion: could be rewritten to `${partnerName} is my ${sex == 'm' ? "wife" : "husband}`

Reply:
Done!

Reply:
Done it!